recipes/libsecret/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libsecret" %}
+{% set version = "0.18.6" %}
+{% set sha256 = "5efbc890ba41a323ffe0599cd260fd12bd8eb62a04aa1bd1b2762575d253d66f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.gnome.org/pub/gnome/sources/libsecret/0.18/libsecret-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Disable-tests-which-require-additional-deps.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    #  https://abi-laboratory.pro/tracker/timeline/libsecret/
+    - {{ pin_subpackage('libsecret', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - intltool
+    - pkg-config
+    - sed  # [osx]
+    # intltool hard pins perl, can't rely on cbc.yaml here
+    - perl 5.22.2.1
```

Comment:
Can't we just say `perl` here, and have `intltool`'s requirement restrict it to this version? Otherwise when/if `intltool` changes the pin then this recipe will break, right?

Reply:
Think that is in bioconda. @bgruening, is there some way to relax the `perl` version constraint of `perl-xml-parser`?

Side note: Don't think we should block the PR on account of this.

Reply:
Hmm...must have missed that. Maybe we need some additional maintainers over there?